docs/development.md

```diff
@@ -92,44 +95,49 @@ following chapters in a certain order:
     - spin up a bare redis container
 
       ```shell-session
-      docker run -d -p 6379:6379 --restart unless-stopped redis:latest
+      $ docker run -d -p 6379:6379 --restart unless-stopped redis:latest
       ```
 
-7.  Install the python dependencies by performing in the src/ directory.
+7.  Install the python dependencies by performing in the root directory:
 
     ```shell-session
-    pipenv install --dev
+    $ cd /path/to/paperless
+    $ pipenv install --dev
     ```
 
-!!! note
+    !!! note
 
-    Make sure you're using python 3.10.x or lower. Otherwise you might
-    get issues with building dependencies. You can use
-    [pyenv](https://github.com/pyenv/pyenv) to install a specific
-    python version.
+        Using a virtual environment is highly recommended.
+        Make sure you're using python 3.10.x or lower. Otherwise you might
+        get issues with building dependencies. You can use
+        [pyenv](https://github.com/pyenv/pyenv) to install a specific
+        python version.
 
 8.  Generate the static UI so you can perform a login to get session
     that is required for frontend development (this needs to be done one
     time only). From src-ui directory:
 
     ```shell-session
-    npm install .
-    ./node_modules/.bin/ng build --configuration production
+    $ cd /path/to/paperless/src-ui
+    $ npm install .
+    $ ./node_modules/.bin/ng build --configuration production
     ```
 
 9.  Apply migrations and create a superuser for your dev instance:
 
     ```shell-session
-    python3 manage.py migrate
-    python3 manage.py createsuperuser
+    $ cd /path/to/paperless/src
+    $ python3 manage.py migrate
```

Comment:
At some point around here, the `pipenv shell` or `pipenv run` needs to be used or these commands won't work